api/src/wfl/log.clj

```diff
@@ -54,10 +54,17 @@
   "A logger to write to standard output"
   (reify Logger
     (-write [logger edn]
-      (-> edn
-          (json/write-str :escape-slash false
-                          :key-fn       key-fn)
-          println))))
+      (let [json-write-str
+            #(json/write-str
+              %
+              :escape-slash false
+              :key-fn       key-fn)
+            to-log
+            (try (json-write-str edn)
+                 (catch Exception x
+                   (json-write-str {:tried-to-log (str edn)
+                                    :cause        x})))]
```

Comment:
I didn't stringify the cause… I'd *hope* that `json/write-str` exceptions could be written as JSON.  Too optimistic?

Reply:
I agree with you.  In a happy future, we get to the backstop only in the rarest of cases.

Reply:
Tested this out and we aren't able to wrap the whole thing in a `pr-str`, the result is not JSON-readable.  But I did stringify the `cause`.

Reply:
> And Throwable … just in case.

Whoops, missed this originally.  Now addressed.